### PR TITLE
[TASK] Revise TCA and use tabs for fields

### PR DESCRIPTION
**Pull request by brotkrueml**
## Description

The TCA has been revised:
- Sort fields into two tabs: General and Access
- Set default of force_ssl to "keep the configured protocol"
- Add descriptions to fields where appropriate
- Use ::class for defining itemsProcFunc
- Adjust text boxes to equal lengths
- Remove not available fields from documentation:
  - Baseurl from Domainrecord
  - Page Id of TypoScript root template

**I have**

- [x] Checked that CGL are followed
- [ ] Checked that the Tests are still working
- [x] Added description to CHANGELOG.md (github-handle is optional)
- [ ] Added tests for the new code

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,16 +3,19 @@
 ## Crawler 9.0.2-dev
 
 ### Added
```
Comment (tomasnorre) [inline CHANGELOG.md:5]: Why did you remove your name? It's ok, just curious why :)

File: Configuration/TCA/tx_crawler_configuration.php [hunk 2]
```diff
@@ -79,7 +81,7 @@
             'label' => 'LLL:EXT:crawler/Resources/Private/Language/Backend.xlf:tx_crawler_configuration.processing_instruction_parameters_ts',
             'config' => [
                 'type' => 'text',
-                'cols' => 40,
+                'cols' => 48,
```
Comment (tomasnorre) [inline Configuration/TCA/tx_crawler_configuration.php:84]: Why is this made bigger? Some default that I haven't respected? It's fine, I just don't know why.

File: Resources/Private/Language/Backend.xlf [hunk 3]
```diff
@@ -27,27 +27,45 @@
...
+                <source>List of processing instructions to send for the request. Processing instructions are necessary for the request to perform any meaningful action, since they activate third party activity.</source>
+            </trans-unit>
             <trans-unit id="tx_crawler_configuration.processing_instruction_parameters_ts">
                 <source>Processing instruction parameters</source>
             </trans-unit>
             <trans-unit id="tx_crawler_configuration.configuration">
                 <source>Configuration</source>
             </trans-unit>
             <trans-unit id="tx_crawler_configuration.base_url">
-                <source>Base URL (leave empty for fetching the Base URL from the Site Configuration)</source>
```
Comment (tomasnorre) [inline Resources/Private/Language/Backend.xlf:37]: I find the hint valid regarding the Base URL, in the future the Base URL should be removed completely, but this is still to soon :)

File: Configuration/TCA/tx_crawler_configuration.php (hunk 2 above)
Comment (brotkrueml) [reply Configuration/TCA/tx_crawler_configuration.php:84]: > Why is this made bigger? Some default that I haven't respected? It's fine, I just don't know why.

The exclude pids ist 48 wide, so I made all this size so it is visually equal

File: Resources/Private/Language/Backend.xlf (hunk 3 above)
Comment (brotkrueml) [reply Resources/Private/Language/Backend.xlf:37]: I removed it from the label and added it as description (new in 9.5):
https://raw.githubusercontent.com/AOEpeople/crawler/a557c926c2aebc160139002c12d22cab9f7d87e1/Documentation/Images/backend_configurationrecord_general.png
Also other fields have now a description for explanation.

File: CHANGELOG.md (hunk 1 above)
Comment (brotkrueml) [reply CHANGELOG.md:5]: I don't like it, once or twice it okay, but not that much ...

Comment (tomasnorre) [reply CHANGELOG.md:5]: But you are helping out a lot right now. Then please consider adding you to the CONTRIBUTERS.md then :)

File: Resources/Private/Language/Backend.xlf (hunk 3 above)
Comment (tomasnorre) [reply Resources/Private/Language/Backend.xlf:37]: Cool, I just learned something new. Thanks.

**Comment by tomasnorre** [comment]
Is this ready to merge if CI is green? from my point of view, the changes look good.

File: CHANGELOG.md (hunk 1 above)
Comment (brotkrueml) [reply CHANGELOG.md:5]: That's okay, done.

**Comment by brotkrueml** [comment]
> Is this ready to merge if CI is green? from my point of view, the changes look good.

Yes, I just forced-pushed me to the contributors.

**Comment by tomasnorre** [comment]
Did you see the Code Style problem in the CI? 

It can be fixed with `.Build/bin/ecs check Classes Configuration Tests --fix`

**Comment by brotkrueml** [comment]
> Did you see the Code Style problem in the CI?
> 
> It can be fixed with `.Build/bin/ecs check Classes Configuration Tests --fix`

Argh, a trailing comma is missing... done
